.github/workflows/main.yml

```diff
@@ -0,0 +1,24 @@
+name: Check PR title and project linting
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: deepakputhraya/action-pr-title@master
+      with:
+        regex: '^(PUB-[0-9]{3,4} [a-zA-Z0-9._\- ]+)|(\[Snyk\].+)$' # Regex the title should match.
```

Comment:
I extended to 4 the digits just to ensure future PRs dont get blocked when we reach 4 digits
we can remove the 3digits if necessary in the future